openapi/definitions.yaml

```diff
@@ -176,6 +180,29 @@ LegalData:
     - sender_mail_from
     - has_attachment
     - message_unique_id
+ThirdPartyData:
+  type: object
+  description: |-
+    Paylod containing all information needed to retrieve and visualize third party message details
+  properties:
+    id:
+      type: string
+      description: Unique id for retrieving third party enriched information about the message
+    original_sender:
+        type: string
+        description: |-
+          Either a ServiceId or a simple string representing the sender name
+        minLength: 1
+    has_attachments:
+      type: boolean
+      default: false
```

Comment:
is useless to set a default for a not required param

Reply:
It's not required, but we are decoding it with `false` as default. You can check related test.